### PR TITLE
[GH-144] Update rpm packaging

### PR DESCRIPTION
**Pull request by Murray-LIANG**
Old rpm package didn't meet the Fedora requirement.

### CODE REVIEW

**Comment by Murray-LIANG** [comment]
@apevec please review the new spec.

**Comment by jealous** [comment]
Please check the CI.  2 out of 3 were failing.

File: utility/rpm_publish/python-storops.spec [hunk 1]
```diff
@@ -126,6 +127,20 @@ Library for managing Unity/VNX systems. Please refer to https://github.com/emc-o
 %exclude %{python3_sitelib}/storops_test*
 %endif
 
+
+%post -n python2-%{pypi_name}
+if [ "$1" -ge 1 ] ; then
+    echo NOTE: If need to managing VNX system, please install Unisphere CLI rpm package first. Contact with Dell/EMC support on that.
```
Comment (apevec) [inline utility/rpm_publish/python-storops.spec]: @Murray-LIANG 
RPM installation might be unattended (e.g. via Satellite) so output from %post scriptlets would be ignored.
Better would be if you would create explicit subpackage python-storops-vnx which requires both python-storops and Unisphere rpm package and document to install that for VNX systems.
That will make missing Unisphere package always clearly visible.

**Comment by apevec** [comment]
@Murray-LIANG Packages for dependencies linked in https://github.com/emc-openstack/storops/releases/tag/r0.4.14 are coming from Fedora buildsystem and might not work on RHEL7. You can find EL7 builds in cbs.centos.org and in RDO repositories (cachez, retryz and persist-queue are being updated to the latest versions from Fedora).

File: utility/rpm_publish/python-storops.spec (hunk 1 above)
Comment (Murray-LIANG) [inline utility/rpm_publish/python-storops.spec]: Thanks for your comments! I updated the spec, added a new spec for storops-vnx. And the new package is uploaded here: https://github.com/emc-openstack/storops/releases/tag/r0.4.14

```
[stack@fedora25 ~]$ rpm -qp --requires /home/stack/rpmbuild/RPMS/x86_64/python2-storops-vnx-0.4.14-1.fc25.x86_64.rpm
NaviCLI-Linux-64-x86-en_US
python2-storops
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1
```

**Comment by Murray-LIANG** [comment]
@apevec Do you have any more comments?

**Pull request merged**